portal/src/graphql/portal/BotProtectionConfigurationScreen.module.css

```diff
@@ -9,3 +9,43 @@
 
   @apply tablet:col-span-full mt-6;
 }
+
+.enabledContent {
+  /* To limit the width of the enabledContent */
+  max-width: 45rem;
+
+  display: flex;
+  flex-direction: column;
+  row-gap: 1rem;
+
+  @apply mt-12;
+}
+
+.columnLeft {
+  grid-column: 1 / span 4;
+}
+
+.columnRight {
+  grid-column: 5 / span 4;
+}
+
+.providerCardContainer {
+  display: grid;
+  grid-template-columns: repeat(8, minmax(0, 1fr));
+  grid-template-rows: auto 1fr;
+
+  column-gap: 0.5rem;
+
+  width: 100%;
+}
+
+.providerCard {
+  padding: 8px 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border-width: 1px;
+  border-style: solid;
+  cursor: pointer;
+  column-gap: 16px;
+}
```

Comment:
I saw you used `rem` as units in previous styles, so lets change these `px` to `rem` too?